library/ntc_reboot.py

```diff
@@ -39,6 +39,11 @@
             - Time in minutes after which the device will be rebooted.
         required: false
         default: null
+    timeout:
+        description:
+            - Time in seconds to wait for the device to come back up.
```

Comment:
Can you say "Time in seconds to wait for the device and API to come back up.  Uses specified port/protocol as defined with port and protocol params."  Of course, I'm hoping that's the case. :)